post/views.py

```diff
@@ -45,4 +58,22 @@ class UserList(generics.ListAPIView):
 class UserDetail(generics.RetrieveAPIView):
     queryset = User.objects.all()
     serializer_class = UserSerializer
+
+def like(request, pk):
+    user = request.user
+    post = Post.objects.filter(id=pk).first()
+    
+    like_obj,created=Like.objects.get_or_create(post=post, user=user)
+
```

Comment:
[trivial]
Extra blank line here.  Linters would catch this. flake8 is the main linter we use.  Look into that and try to integrate it into your workflow.  If you get stuck, don't be afraid to ask Pavel or myself for help :)